cuebot/src/main/resources/conf/ddl/postgres/migrations/V1__Initial_schema.sql

```diff
@@ -17,6 +17,7 @@ CREATE TABLE frame_history (
     int_mem_reserved BIGINT DEFAULT 0 NOT NULL,
```

Comment:
We will definitely want to do this as a new migration instead of updating an existing migration.

That will also necessitate a minor version bump.